18 Normalizr/src/pages/student-list/components/studentRow.tsx

```diff
@@ -7,6 +7,13 @@ interface Props {
 }
 
 export const StudentRowComponent = (props: Props) => {
+
+  const showCountryName = () => {
```

Comment:
Don't say `show` when you mean `get` ;)
But the whole comparison with 0 seems a bad smell.